docker/templates/python/cookiecutter.json

```
@@ -0,0 +1,11 @@
+{
+    "directory_name": "python-template",
+    "app_name": "app",
```

This might make it clearer that the value should be a slug:

```suggestion
    "app_name": "my-new-app",
```